confluent_kafka/schema_registry/__init__.py

```diff
@@ -16,13 +16,7 @@
 # limitations under the License.
 #
 
-from .config import SchemaRegistryConfig, RegistrySerializerConfig
-from .error import SchemaRegistryClientError
-from .schema_registry_client import SchemaRegistryClient, CompatibilityType
-from .serdes import RecordNameStrategy,\
-    TopicNameStrategy, TopicRecordNameStrategy
-
-__all__ = ['CompatibilityType',
-           'SchemaRegistryClient', 'SchemaRegistryClientError',
-           'SchemaRegistryConfig', 'RecordNameStrategy',
-           'RegistrySerializerConfig', 'TopicNameStrategy', 'TopicRecordNameStrategy']
+# from .schema_registry_client import SchemaRegistryClient, CompatibilityType
```

Comment:
I'll revisit this at the very end to ensure everything feeds into the main library __init__ file properly.